Tests/WebUITests/WebViewProxyTests.swift

```diff
@@ -1,9 +1,132 @@
+@preconcurrency import Combine
 import Foundation
 import Testing
 
 @testable import WebUI
 
 struct WebViewProxyTests {
+    @MainActor @Test
+    func property_binding_title() async throws {
+        let sut = WebViewProxy()
+        let webViewMock = Remakeable {
+            EnhancedWKWebViewMock(title: "dummy") as EnhancedWKWebView
+        }
```

Comment:
Is the current implementation sufficient to ensure the behavior we want to verify this time?
I had the impression that it might be more appropriate to set the value after initialization, like in the example below.
What do you think?
```suggestion
        let webViewMock = Remakeable {
            EnhancedWKWebViewMock() as EnhancedWKWebView
        }
        (webViewMock.wrappedValue as? EnhancedWKWebViewMock).title = "dummy"
        // It might be more appropriate to wrap this in a Task
        // Task { (webViewMock.wrappedValue as? EnhancedWKWebViewMock).title = "dummy" }
```

Reply:
That may be so.
I was avoiding this method because I can't access the property without casting it in `EnhancedWKWebViewMock`. But I thought the way you suggested was better.

Reply:
Changed in [660c10a](https://github.com/cybozu/WebUI/pull/59/commits/660c10a24e9f08d11b5e80c20517f48d672d0441)


I gave up because the test did not pass when I executed `yield()` if I enclosed the line of property assignment in a `Task`.

```swift
Task {
    (webViewMock.wrappedValue as? EnhancedWKWebViewMock)?._title = "dummy"
}
await Task.yield
```

Reply:
I also couldn't figure out how to get the tests running properly when they're inside a `Task`.
@elmetal, do you have any ideas?

Reply:
@b1ackturtle @Kyome22 
I thought the following code would work well, but the test failed. It seems like `sut.$title.values` is behaving unexpectedly.

```swift
        Task {
            try await Task.sleep(for: .seconds(1))
            (webViewMock.wrappedValue as? EnhancedWKWebViewMock)?.title = "dummy"
        }
        print(webViewMock.wrappedValue.title) // nil
        let actual = try await waitForValue(
            in: sut.$title.values,
            equalsTo: "dummy",
            timeout: .seconds(3.5)
        )
        print(webViewMock.wrappedValue.title) // Optional("dummy")
        #expect(actual)
```